src/components/CustomToast/CustomToast.tsx

```diff
@@ -0,0 +1,136 @@
+import "./CustomToast.scss";
+import {FC, useEffect, useRef, useState} from "react";
+import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
+import {ToastTypes} from "utils/Toast";
+import classNames from "classnames";
+
+export interface CustomToastProps {
+  title: string;
+  message?: string;
+  hintMessage?: string;
+  hintOnClick?: () => void;
+  buttons?: string[];
```

Comment:
you are able to add as many buttons as you want, however these look weird and also don't have any functionality. Maybe changing this to `primaryButton?` and `secondaryButton?` or just cutting off the extra buttons would solve this?